projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java

```diff
@@ -531,6 +531,20 @@ public String getAnswer(
     return answer;
   }
 
+  /**
+   * Return a {@link SortedSet SortedSet} contains all subdirectories under the path {@code
```

Comment:
A Container is not a sorted set -- it should be a `Container`. Please make a proper java object that represents a container. One field it may have is `testrigs`, which could be an ordered list or set of testrigs.